doc/pages/expansions.asciidoc

```diff
@@ -48,9 +48,9 @@ for full details). For example:
 
 * `echo "x%val{session}x"` echoes the current session ID, surrounded by `x`
 
-* `echo %{%val{session}}"` echoes the the literal text `%val{session}`
+* `echo "%{%val{session}}"` echoes the the literal text `%val{session}`
```

Comment:
I think both these quotes are not necessary.

`%val{session}` is already quoted by being surrounded by `%{` and `}`.

Reply:
Yeah, the point of this example is to show that a single layer of %{} quoting will prevent the contents from being expanded. This is a case of too *many* quotation marks, not missing quotation marks.